.github/workflows/dangerci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
```

Comment:
We don't need setup-node anymore, we can use the .nvmrc file as la https://github.com/Automattic/jetpack/pull/18213/files#diff-3625957935f73eaca5408fdda40fce0a719a8551f975cefdc9249efa2cd840d3R22 , but we can pick that up in a follow-up PR since it doesn't negatively impact building.

Reply:
At a quick glance through its documentation, it seems like nvm does more or less the same thing that setup-node does. It may be more flexible about things like asking for version "lts" instead of "12" or using non-official mirrors, but I don't think we need either of those things. On the down side, it seems it's per-shell, so I think we'd have to re-run it in every step that will be executing node (i.e. more boilerplate).

And setup-node@v2 may be better about taking advantage of

> The hosted images have been updated with the latest of each LTS from v8, v10, v12, and v14.

(quoted from https://github.com/actions/setup-node#v2)

Reply:
> On the down side, it seems it's per-shell, so I think we'd have to re-run it in every step that will be executing node (i.e. more boilerplate).

We should be able to set a default so it only has to be set at the start. The advantage in my mind is whenever we do bump to node 14, we only need to update .nvmrc and not all of the actions to match the version.

But in either case, we can dive into it in a future PR. No need to get sidetracked on it now.